.travis.yml

```diff
@@ -7,6 +7,18 @@ env:
 
 matrix:
   include:
+  - os: osx
+    name: "Check Shell Scripts"
+    osx_image: xcode10.2
+    compiler: clang
+
+    script:
+      - HOMEBREW_NO_AUTO_UPDATE=1 brew install shellcheck
+      # Check Parent Dir (No *.sh files curently. Change "*.tool" to "*.{sh,tool}" if any is added later)
+      - shellcheck /home/travis/build/acidanthera/OpenCorePkg/*.tool
```

Comment:
Use `$(pwd)` if you want full path. We should not rely on travis internal paths.